app/domain/authentication/authn_oidc/get_conjur_oidc_token/authenticator.rb

```diff
@@ -3,62 +3,59 @@
 module Authentication
   module AuthnOidc
     module GetConjurOidcToken
-      class OIDCAuthenticationError < RuntimeError; end
-
-      class Authenticator
-        extend CommandClass::Include
-
-        command_class(
-          dependencies: {
-          },
-          inputs: %i(input oidc_id_token_details)
-        ) do
+      class OIDCAuthenticationError < RuntimeError;
+      end
 
-          def call
-            validate_id_token_claims
-            validate_user_info
-            # TODO: validate_nonce
-            true
-          end
+      Authenticator = CommandClass.new(
+        dependencies: {
```

Comment:
This one does actually have a dependency, it's just hardcoded at the moment.  On line 43, `Resource` is a dep.  That should passed in here, which then allows this to be unit tested without hitting the actual db (via a mock for Resource)